### ISSUE TITLE
LoRA beseline parameter

### ISSUE THREAD

**Issue by jypppppp** [TsinghuaC3I/SoRA#4]
Hi,

Thanks for your good work!

Can you clarify what is learning rate,bsz and epochs for baseline LoRA experiments among different datasets

Kind regards,

Jason

**Comment by telxt** [comment]
<html><body>
<p>
	Thank you for your interest in our work! The hyper-parameters of LoRA is listed in the following:<br>
</p>

Dataset | lr | epoch
-- | -- | --
CoLA | 8e-5 | 20
SST-2 | 1e-4 | 10
MRPC | 1e-4 | 20
QQP | 3e-4 | 10
STS-B | 1e-4 | 20
MNLI | 3e-4 | 10
QNLI | 3e-4 | 10
RTE | 1.2e-3 | 50


<br>
<p>
	And the seed list is {0, 21, 42, 81, 100}, the batch_size is 8.<br>
</p>
<p>
	I hope my response helps you.</p>
</body>
</html>

**Comment by ouxinwei111** [comment]
Hi, I was wondering if you use the same learning rate for all the rank settings. Looking forward to your help :)
